conda-recipes/impyla/meta.yaml

```diff
@@ -25,6 +26,7 @@ requirements:
     - thrift_sasl
     - bitarray
     - thriftpy
+    - sasl >=0.2.1
```

Comment:
Actually according to https://github.com/cloudera/impyla/blob/5ff16c84704d6b73023e8d71e5fe07ee0a844dfb/setup.py#L35 these are hard deps.  Guess we can change them in impyla.  Currently they are required still afaik.